house_rules/Avrae.md

```diff
@@ -0,0 +1,27 @@
+# Avrae
+
+:::note Out of Character
+Avrae is the internal and external will of the universe. Pray they take pity on you.
+:::
+
+Avrae is a discord dnd-plugin used to roll dice and ability checks in the discord server.
+
+## Setting up
+
+Use ```!import [dndbeyond character hyperlink]``` where the [dndbeyond character hyperlink] is your DNDBeyond character's share hyperlink. This loads your character into Avrae. Once the character is imported, you can use avrae commands. 
+
+Below is an example of where the share link is. 
+![Avrae_Example](/img/avrae_share_example.png)
+
+You can do ```!rollstats``` to roll stats for a new character.
+
+## Common Rolls
+
+- ```!char [character name]``` changes your active character to [character name]. This requires loading your characters before hand
```

Comment:
```suggestion
- ```!char [character name]``` changes your active character to [character name]. This requires loading your characters beforehand.
It will try to match from among your characters, so if your characters are `Bob` and `Faerun`, you can shorthand Faerun to `!char Fa`.
```